mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb

```diff
@@ -73,15 +97,23 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
         # https://cloud.google.com/run/docs/configuring/cpu-allocation
         "run.googleapis.com/cpu-throttling" = false
       }
+      # [END cloudrun_service_configuration_max_instances]
+      # [END cloudrun_service_configuration_min_instances]
 
+      # [START cloudrun_service_configuration_labels]
       # Labels
       # https://cloud.google.com/run/docs/configuring/labels
       labels = {
         foo : "bar"
         baz : "quux"
       }
+      # [END cloudrun_service_configuration_max_instances]
+      # [END cloudrun_service_configuration_min_instances]
```

Comment:
This tag starts on line `82` and ends online `111` and `116`.

Reply:
Thanks! Fixed.